src/Controller/Component/FlashComponent.php

```diff
@@ -20,14 +23,21 @@
  * @method void error(string $message, array $options = []) Set a message using "error" element
  * @method void warning(string $message, array $options = []) Set a message using "warning" element
  * @method void info(string $message, array $options = []) Set a message using "info" element
+ * @method void transientSuccess(string $message, array $options = []) Set a message using "success" element.
+ *                                                                      These flash messages that are not saved (only available for current view)
```

Comment:
saved => persisted across requests

You can also move this to the front and have 3 whitespace indentation.